### ISSUE TITLE
Todo example: add support for the router

### ISSUE THREAD

**Issue by vicb** [angular/angular#2100]
In the Todo examples, the links (completed, all) are ineffective. 

IIRC the implementation was pending on the router being available.

I think it could be updated now ?

/cc @davideast @btford

**Comment by mhevery** [comment]
@btford update?

**Comment by mhevery** [comment]
This is obsolete, and no longer relevant / actionable. To keep our issues clean, we are aggressively closing them.
